pupil-spa/src/app/virtual-keyboard/virtual-keyboard.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="question-container">
+  <form id='js-question-form' action="/check/question/{{questionNum}}" method="POST" autocomplete="off">
```

Comment:
@jon-shipley Essentially the action is not part of this story, I left it as it was but I've removed it as per your suggestion.